Dockerfile-check

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.7.6
+
+ENV GOPATH=/go
+
+WORKDIR /go/src/github.com/contiv/netplugin/
+
+RUN go get github.com/tools/godep github.com/aktau/github-release \
```

Comment:
Is `github.com/aktau/github-release` here intentionally? 😄

Reply:
was copied, removed!